build/syncset.go

```diff
@@ -436,20 +450,20 @@ func main() {
 		if sliceContains(hook().Name(), skip) {
 			continue
 		}
-		if len(onlyInclude) > 0 {
-			if sliceContains(hook().Name(), onlyInclude) {
-				encoded = append(encoded, runtime.RawExtension{Raw: encode(createValidatingWebhookConfiguration(hook()))})
-			}
+		if len(onlyInclude) > 0 && !sliceContains(hook().Name(), onlyInclude) {
 			continue
 		}
-		// can't use RawExtension{Object: } here because the VWC doesn't implement DeepCopyObject
-		encoded = append(encoded, runtime.RawExtension{Raw: encode(createValidatingWebhookConfiguration(hook()))})
+		templateResources.Add(hook().SyncSetLabelSelector(), runtime.RawExtension{Raw: encode(createValidatingWebhookConfiguration(hook()))})
 	}
 	// TODO(lseelye): Until 4.3, place the DaemonSet after the
 	// ValidatingWebhookConfigurations are added to the `encoded` list so that the
 	// initContainer will work (see createDaemonSet and
 	// createValidatingWebhookConfiguration)
-	encoded = append(encoded, runtime.RawExtension{Object: createDaemonSet()})
+	templateResources.Add(metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			"api.openshift.com/managed": "true",
+		},
```

Comment:
Can we move this above the creation of the `createValidatingWebhookConfiguration` and use the `utils.DefaultLabelSelector()` call here?

Reply:
done